group_vars/location_gubener/owm.yml

```diff
@@ -1,4 +1,4 @@
 ---
 location_nice: Gubener37
-latitude: 52.510169
-longitude: 13.450146
+latitude: 52.50945371664022
+longitude: 13.447127850710801
```

Comment:
What is happening here? Can you rebase?

Reply:
The one commit should already be merged:
https://github.com/Freifunk-Spalter/bbb-configs/commit/6bc610d964b55cc9b14f44d909959b6fbb1679e1